content/attacks/posts/2023-05-01-Level-Finance.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2023-05-01
+attacks/posts/categories: "Protocol exploit"
+title: "Level Finance Hacked for $1.1 Million in LVL Tokens"
+---
+
+## Summary
+
+On May 1, 2023, Level Finance, a decentralized finance (DeFi) protocol, was hacked for $1.1 million in LVL tokens. The attacker [exploited a vulnerability in the protocol's Referral Controller Contract](https://www.halborn.com/blog/post/explained-the-level-finance-hack-may-2023). 
+
+
+## Attackers
+
+The identity of the attacker is unknown.
+
+brc-20:
```

Comment:
Delete the line, or rewrite. ERC20 and BEP20 is a token standards, BSC is a chain.

Reply:
of course, it was an accidental typo